modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
```

Comment:
@anarnold97 and I thought it would be useful to add a link to FAQ in the release notes https://access.redhat.com/articles/5456281
probably at the top

Reply:
ack, the link is now added. 
@weshayutin, are there any other suggestions for the added release notes or does it look good?